modules/core/src/main/java/org/apache/synapse/unittest/Assertor.java

```diff
@@ -0,0 +1,459 @@
+/*
+ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.synapse.unittest;
+
+import javafx.util.Pair;
+import org.apache.http.Header;
+import org.apache.http.HttpResponse;
+import org.apache.http.util.EntityUtils;
+import org.apache.log4j.Logger;
+import org.apache.synapse.MessageContext;
+import org.apache.synapse.core.axis2.Axis2MessageContext;
+import org.apache.synapse.unittest.testcase.data.classes.AssertEqual;
+import org.apache.synapse.unittest.testcase.data.classes.AssertNotNull;
+import org.apache.synapse.unittest.testcase.data.classes.TestCase;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+import static org.apache.synapse.unittest.Constants.INPUT_PROPERTY_AXIS2;
+import static org.apache.synapse.unittest.Constants.INPUT_PROPERTY_BODY;
+import static org.apache.synapse.unittest.Constants.INPUT_PROPERTY_CONTEXT;
+import static org.apache.synapse.unittest.Constants.INPUT_PROPERTY_TRANSPORT;
+
+
+/**
+ * Class responsible for the validation of testing with expected results.
+ */
+class Assertor {
+
+    private static Logger logger = Logger.getLogger(Assertor.class.getName());
```

Comment:
Rename `logger` to `log` because `log.info` looks nice than `logger.info`

Reply:
Done